### ISSUE TITLE
Filter feature on TVshows

### ISSUE THREAD

**Issue by jungwonOh0920** [jungwonOh0920/tmdb#88]
needs to implement the filter feature on tv page.

**Comment by Ackpace** [comment]
Hello!

I find this feature interesting and would like to work on it.

Can you please assign this to me?

**Comment by jungwonOh0920** [comment]
> Hello!
> 
> I find this feature interesting and would like to work on it.
> 
> Can you please assign this to me?

Hey. Sorry just checked this. This is my humble personal project to teach myself about stuff. nothing fancy. If you still want to fix this problem, yes you can definitely do that. Still in?

**Comment by Ackpace** [comment]
I understand you. I am also learning this way but I would like to finish this task as I am currently working on it. Can you set this task to "open" again and assign me this task ? because then I would like to do a merge as well.

**Comment by jungwonOh0920** [comment]
Yes Sure. Make sure to pull again since I already push my code :)

**Comment by jungwonOh0920** [comment]
I have filter on the left and contents on the right. The issue to fix for this ticket is the contents need to be displayed based on the filter status. Let me know if you have any questions 👍 

![Image](https://github.com/jungwonOh0920/tmdb/assets/110989864/c6db80b0-64cf-4731-8208-0f0d41eae6c9)

**Comment by Ackpace** [comment]
All right. Thank you.
